packages/driver/cross-origin-testing.md

```diff
@@ -73,17 +78,76 @@ Browser automation APIs and the proxy play a small but critical role in facilita
 
 The proxy intercepts http requests from all sources. In order to detect cross-origin navigation of **AUT**, it’s necessary to know that a request came specifically from the **AUT frame** and not **top**, a nested iframe, or elsewhere. To achieve this, we use the browser automation APIs to add a `X-Cypress-Is-AUT-Frame` header to any requests from the AUT.
 
-Allowing the proxy to know if a request is from the **AUT** enables us to do two things when it recognizes that it’s not the **primary origin**:
-
-- Delay the response. This allows us to communicate with the **primary driver** to set up the **spec bridge** and run the user’s callback function via **cy.origin()**. Then the driver notifies the proxy that it can allow the response through. This enables users to listen to the `window:before:load` event in the callback function, since the page will not load until after any such listeners are set up.
-- Inject code into the request that’s tailored to cross-origin testing.
+Allowing the proxy to know if a request is from the **AUT** enables us to recognizes that it’s not the **primary origin** and inject code into the request that’s tailored to cross-origin testing.
```

Comment:
Do we want to be more specific here and say that it isn't the superDomainOrigin since sub domain navigations have `main` injection?

Reply:
Might be getting in the weeds a little bit for the purposes of communicating the role of the proxy/automation. 

I think we should revisit this doc once decisions are settled around strict same-origin spec bridges. Then we can clarify any language around cross-origin and cross-superDomainOrigin and elucidate any differences between primary-origin navigation and navigation within `cy.origin()`.